.changeset/cuddly-stingrays-jog.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I dont think you need changeset. Just remove it

Reply:
I've got CI failing because of that 🤔 
![CleanShot 2024-01-30 at 13 17 04@2x](https://github.com/saleor/apps/assets/9116238/042873a0-4e53-4b69-a9c5-f768e47256bc)

https://github.com/saleor/apps/actions/runs/7708234070/job/21006898286?pr=1182

Reply:
there is label for that, added

Reply:
https://github.com/saleor/apps/pull/1183/files